README.rst

```diff
@@ -23,7 +23,7 @@ If you'd like to join them, please consider `sponsoring Twisted's <https://docs.
 What is this?
 -------------
 
-Twisted is an event-based framework for internet applications, supporting Python 3.6+.
+Twisted is an event-based framework for internet applications, supporting Python 3.8+.
```

Comment:
Maybe something like this... so that in the future, we should only deal updating pyproject.toml


```suggestion
Twisted is an event-based framework for internet applications, supporting Python 3.
Check the `requires-python` value from `pyproject.toml <pyproject.toml>`_ for the minimum supported Python version.
```

Reply:
I was just looking to make use of the effort that pointed out this error.  This is also fine by me.  Though I might just taking out the supporting python 3 bit entirely.  I also don't really think the explanation of where to check the minimum version is critical.  Then again, I've never liked duplicating such standard python project instructions in every project's readme.  `:]`

```suggestion
Twisted is an event-based framework for internet applications.
```

I'm fine either way.